cloud/aws/bin/restoredb.py

```diff
@@ -0,0 +1,197 @@
+import os
+import ipaddress
+import sys
+import tempfile
+import shlex
+import subprocess
+import textwrap
+import urllib.request
+from pathlib import Path
+from time import sleep
+from datetime import datetime
+
+from cloud.aws.templates.aws_oidc.bin import resources
+from cloud.aws.templates.aws_oidc.bin.aws_cli import AwsCli
+from cloud.shared.bin.lib import terraform
+from cloud.shared.bin.lib.config_loader import ConfigLoader
+from cloud.shared.bin.lib.print import print
+
+
+class Color:
```

Comment:
**suggestion:** Since we're using this in other places, we could consider pulling it out into a separate file?